libs/database/src/lib/schemas/user.schema.ts

```diff
@@ -191,11 +194,15 @@ UserSchema.statics.toAuthor = (self: User | UserDocument) =>
   } as Author);
 
 UserSchema.methods.toUserResponse = async function (
-  passwordNotSet = false,
-  blocked?: boolean,
-  blocking?: boolean,
-  followed?: boolean
+  option = {
+    passwordNotSet: undefined,
+    blocked: undefined,
+    blocking: undefined,
+    followed: undefined,
+    balance: undefined,
+  } as UserResponseOption
 ) {
+  const { passwordNotSet, blocked, blocking, followed, balance } = option;
```

Comment:
```suggestion
  { passwordNotSet, blocked, blocking, followed, balance } = { } as UserResponseOption
) {
```

or convert type `UserResponseOption` to class and use

```suggestion
  { passwordNotSet, blocked, blocking, followed, balance } = new UserResponseOption()
) {
```